### PR TITLE
Add private registry info

### PR DESCRIPTION
**Pull request by johnsmartco**
**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Comment by johnsmartco** [comment]
Hi @jeffreyscarpenter.  FYI, this is the pending PR for the Private Registries topic.  After reviews, once published on the site, the URL (for your 1.3.0 blog) will be:

https://docs.k8ssandra.io/tasks/manage/private-registries/

I'll ping you when it's there.  Thanks Jeff and John @jsanda.

File: docs/content/en/tasks/manage/private-registries/_index.md [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+| `cassandra`                                              | Runs the Management API and Cassandra&reg; |
+| `cassandra.configBuilder`                                | An init container that provisions Cassandra configuration files such as cassandra.yaml |
+| `cassandra.jmxCredentialsConfig` | An init container that configures authentication for remote JMX access |
+| `cassandra.loggingSidecar` | A container that tails Cassandra's system.log file |
+| `stargate` | Runs Stargate, which is an API for interacting with Cassandra |
+| `stargate.waitForCassandra` | An init container that waits until all the Cassandra pods are ready |
+| `reaper` | Run Reaper, which provides repair functionality for Cassandra data |
+| `medusa` | Runs the medusa container that performs backups, and the `medusa-restore` init container that performs restores |
+| `cleaner` | A Helm pre-delete hook that deletes the CassandraDatacenter |
+| `client` | Used in the `crd-upgrader pre-upgrade` to apply Custom Resource Definition (CRD) updates |
```
Comment (jsanda) [inline docs/content/en/tasks/manage/private-registries/_index.md:96]: I would change `crd-upgrader pre-upgrade` to `crd-upgrader` pre-upgrade hook

File: docs/content/en/tasks/manage/private-registries/_index.md [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+| `cass-operator` | Cass Operator |
+| `reaper-operator` | Reaper Operator |
+| `medusa-operator` | Medusa Operator |
+| `kube-prometheus-stack.prometheus-operator` | Prometheus Operator |
+| `kube-prometheus-stack.prometheus-operator.prometheusSpec` | Prometheus to collect Cassandra cluster, OS, and node metrics |
+| `grafana` | Grafana to visualize the metrics collected by Prometheus |
+
+{{% alert title="Tip" color="success" %}}
+A general note: Operators are software extensions to Kubernetes that make use of custom resources to manage applications and their components. Thus, for example, "Reaper Operator" deploys and configures Reaper. "Reaper" itself manages the actual Cassandra repair operations. Similarly, "Prometheus Operator" deploys and configures Prometheus. "Prometheus" itself manages the actual collection of relevant OS / Cassandra metrics. "Medusa Operator" deploys and configures Medusa. "Medusa" itself manages the backup/restore of Cassandra data.
```
Comment (jsanda) [inline docs/content/en/tasks/manage/private-registries/_index.md:105]: >"Medusa Operator" deploys and configures Medusa. "Medusa" itself manages the backup/restore of Cassandra data.

Medusa Operator is a bit atypical compared to the other operators in that it does not deploy and configure Medusa. It configures and orchestrates the backup and restore operations.

**Review by jsanda — CHANGES_REQUESTED** [review]

File: docs/content/en/tasks/manage/private-registries/_index.md (hunk 1 above)
Comment (johnsmartco) [reply docs/content/en/tasks/manage/private-registries/_index.md]: Updated in subsequent commit - thanks John.

File: docs/content/en/tasks/manage/private-registries/_index.md (hunk 2 above)
Comment (johnsmartco) [reply docs/content/en/tasks/manage/private-registries/_index.md]: Updated in subsequent commit - thanks John.

**Review by jsanda — APPROVED** [review]

**Pull request merged**